sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientLiveTests.cs

```diff
@@ -2059,6 +2227,11 @@ public async Task StartRecognizeCustomFormsWithLabelsAndSelectionMarks(bool incl
             var name = "AMEX_SELECTION_MARK";
             Assert.IsNotNull(form.Fields[name]);
             Assert.AreEqual(FieldValueType.SelectionMark, form.Fields[name].Value.ValueType);
+
+            // If this assertion is failing after a recent update in the generated models, please remember
+            // to update the manually added FieldValue_internal constructor in src/FieldValue_internal.cs if
+            // necessary. The service originally returns "selected" and "unselected" as lowercase strings,
+            // but we overwrite these values there.
```

Comment:
Consider adding https://github.com/Azure/azure-sdk-for-net/issues/17814 saying that once that is fixed we should delete this?